### ISSUE TITLE
WholeBody 2D-to-3D-pose lifting

### ISSUE THREAD

**Issue by Jiaqi-zhang** [open-mmlab/mmpose#1697]
Thanks. When I tried to convert the Coco-WholeBody 2D Pose to 3D, I got the following error message. 

_“unsupported conversion between Body3DH36MDataset and TopDownCocoWholeBodyDataset”_

What do I need to do to implement this transition process?

demo command:
`
python demo/body3d_two_stage_video_demo.py \
    demo/mmdetection_cfg/faster_rcnn_r50_fpn_coco.py \
    pretrained_models/faster_rcnn_r50_fpn_1x_coco_20200130-047c8118.pth \
    configs/wholebody/2d_kpt_sview_rgb_img/topdown_heatmap/coco-wholebody/hrnet_w48_coco_wholebody_384x288_dark_plus.py \
    pretrained_models/hrnet_w48_coco_wholebody_384x288_dark-f5726563_20200918.pth \
    configs/body/3d_kpt_sview_rgb_vid/video_pose_lift/h36m/videopose3d_h36m_243frames_fullconv_supervised_cpn_ft.py \
    pretrained_models/videopose_h36m_243frames_fullconv_supervised_cpn_ft-88f5abbb_20210527.pth \
    --video-path view.mp4 \
    --out-video-root vis_results \
    --rebase-keypoint-height
`

**Comment by ly015** [comment]
Please note that our pose lifting models are trained on Human3.6M data. You can manually convert 2D keypoints in other formats (e.g. COCO) into the **h36m format**, and feed them into the model to get 3D keypoints in the **h36m format**. Usually COCO 2D keypoint is enough to get h36m 2D keypoints (and it's already supported in the code), while COCO-wholebody keypoints could be redundant for this purpose.

To manually support converting other 2D keypoints into h36m format for pose lifting, you can add codes [here](https://github.com/open-mmlab/mmpose/blob/master/demo/body3d_two_stage_video_demo.py#L28).

**Comment by Jiaqi-zhang** [comment]
> Please note that our pose lifting models are trained on Human3.6M data. You can manually convert 2D keypoints in other formats (e.g. COCO) into the **h36m format**, and feed them into the model to get 3D keypoints in the **h36m format**. Usually COCO 2D keypoint is enough to get h36m 2D keypoints (and it's already supported in the code), while COCO-wholebody keypoints could be redundant for this purpose.
> 
> To manually support converting other 2D keypoints into h36m format for pose lifting, you can add codes [here](https://github.com/open-mmlab/mmpose/blob/master/demo/body3d_two_stage_video_demo.py#L28).

thanks a lot

**Comment by chemingmei** [comment]
I would like to ask whether such conversion will lead to a decline in the quality of 2D data. I've found that RTMPose outperforms the CPN model on the COCO dataset. However, when I trained my own 3D model using 2D pose data in Human3.6M format, which was converted from the COCO - format data generated by RTMPose and the data from the CPN model respectively, the 2D data input provided by the CPN model actually resulted in a lower MPJPE. Does this mean that if I want RTMPose to perform better, I need to retrain it using the Human3.6M - format dataset?
